src/main/resources/db/migration/V013_Add_new_fields.sql

```diff
@@ -0,0 +1,10 @@
+ALTER TABLE envelopes
```

Comment:
If fields are mandatory then they should be not null ? We will also need to populate default values if field are mandatory as part of migration.